ci-operator/step-registry/openshift-qe/installer/bm/deploy/openshift-qe-installer-bm-deploy-commands.sh

```
@@ -86,6 +86,14 @@ image_type: "minimal-iso"
 reset_idrac: $RESET_IDRAC
 EOF
 
+# Append ocp_inventory_override if enabled
+if [[ $OCP_INVENTORY_OVERRIDE == "true" ]]; then
+   OCP_INVENTORY_PATH=$(cat ${CLUSTER_PROFILE_DIR}/ocp_inventory_path)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)